ClientApp/KachnaOnline/src/app/models/user.model.ts

```diff
@@ -0,0 +1,11 @@
+// user.model.ts
+// Author: David Chocholatý
+
+export class User {
+  name: string = "";
+  email: string = "";
+  nickname: string = "";
+  gamificationConsent: boolean = false;
+  cardNumber: string = "";
```

Comment:
This should rather be called `cardCode` (the validation regex for this is `[A-Z]{5}`).

Reply:
Thank you. I was intending to ask you about that. I will add the constraints now, then.